design/FY2018/NFP-Disable Self-Shading Effect.md

```diff
@@ -0,0 +1,75 @@
+Disable Self-Shading Effect
+================
+**Tianzhen Hong, Xuan Luo, LBNL**
+
+**Original: November 2, 2017**
+
+# Justification for New Feature 
+
+Building energy modeling for LEED certification requires to disable self-shading effect in the baseline model, which gives credit to the proposed building with self-shading. For example, an L-shape building that is oriented correctly can provide significant self-shading and ASHRAE Standard 90.1 wants to reward that design choice. The proposed new feature will allow users to disable self-shading effect in EnergyPlus.
```

Comment:
I still am not sure this is the correct application for a multiple-building model.  I see two primary use-cases for multiple-building models.

1.  All of the buildings are new and should get credit for their geometry.  In this case, it would seem that the placement of the buildings relative to each other should also get credit.  That would say that all self-shading, even between buildings should be excluded.

2.  Some of the buildings are existing, and one ore more new buildings are being added to the campus.  In this case, the existing buildings should self-shade so they don't impact the shading savings of the new buildings.  The surrounding buildings aren't part of the proposed design.

The proposed input objects cannot do case 2.